examples/uccf12/uccf12.cpp

```diff
@@ -78,38 +79,89 @@ try_main() {
 
   // single commutator, needs symmetrization
   {
-    auto h = H();
+    auto h = H(false);
     auto r = R12(gg_space);
-    auto hr_comm = do_wick( ex<Constant>(2) * (h*r - r*h) );  // this assumes symmetrization includes 1/2
+    /*auto hr_comm = do_wick( ex<Constant>(2) * (h*r - r*h) );  // this assumes symmetrization includes 1/2
```

Comment:
is this meant to be removed? Commented out code should be removed, unless used as documentation of something